javascript/central-login-oidc/.env.example

```diff
@@ -0,0 +1,6 @@
+SCOPE=$SCOPE // For using "PINGONE" as the server type, the `revoke` scope must be set as well.
+TIMEOUT=$TIMEOUT
```

Comment:
We say 3000 to 5000 is good, perhaps we just choose one for them here?

Reply:
This is very debatable on what values they should need. I added a few more details